pkgs/top-level/python-packages.nix

```diff
@@ -56,7 +56,9 @@ let
   # See build-setupcfg/default.nix for documentation.
   buildSetupcfg = import ../build-support/build-setupcfg self;
 
-  fetchPypi = callPackage ../development/interpreters/python/fetchpypi.nix { };
+  # fetchPypi -> fetchFromPyPI
```

Comment:
```suggestion
```
I can read that below, no need for a comment.

Reply:
> Using the new fetcher will probably break nix-update, nixpkgs-update and nix-prefetch-url.

How?
This is not a "new fetcher", it is the same with a new name. Do the updaters rely on the name of the function?


> This is pretty clear from the line below, no need to comment it.

Okay!

Reply:
nix-prefetch-url should be fine. AFAIK, it only works with the nix store and curl.

`nix-update` might be sensitive to that term. But it's not like we are rolling it out immediately

Reply:
Sorry I accidently edited your message instead of replying @SuperSandro2000 

I wanted to say that there are other possible spellings like `fetchFromPypi`, I'm not sure about the change in capitalization